RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1277,6 +1277,19 @@ boolean L11_hiddenCity()
 			return false;
 		}
 	}
+	
+	//can we handle this zone?
+	if(!acquireHP())	//try to restore HP to max.
+	{
+		return false;		//could not heal HP. we should go do something else first
+	}
+	int weapon_ghost_dmg = numeric_modifier("hot damage") + numeric_modifier("cold damage") + numeric_modifier("stench damage") + numeric_modifier("sleaze damage") + numeric_modifier("spooky damage");
+	if(weapon_ghost_dmg < 20 &&				//we can not rely on melee/ranged weapon to kill the ghost
+	!acquireMP(30))							//try getting some MP, relying on a spell to kill them instead. TODO verify we have a spell
```

Comment:
Given the minimum requirements to use autoscend, most users are likely to have at least 1 spell, likely just challenge paths that would be throwing an issue?

Reply:
Yea. basically why I figured it is ok to leave it as a todo.